lib/cli.js

```diff
@@ -118,6 +118,10 @@ exports.run = () => {
 		throw new Error(colors.error(figures.cross) + ' Watch mode is not available in CI, as it prevents AVA from terminating.');
 	}
 
+	if (hasFlag('--concurrency') || hasFlag('-c')) {
```

Comment:
This is not the correct way to check. AVA can be run with either `ava --concurrency=1` or `ava --concurrency 1`.